src/main/webapp/projectContainer/ProjectContainer.js

```diff
@@ -154,7 +153,8 @@ export const ProjectContainer = hh(class ProjectContainer extends Component {
                 key: "comments",
                 title: "Comments",
               }, [
-                  h(Fragment, {}, [Comments({
+                  h(Fragment, {}, [
```

Comment:
another not needed Fragment here

Reply:
I've removed all these fragments, they all seem to be unnecessary.